src/prop/theory_proxy.cpp

```diff
@@ -293,6 +293,14 @@ void TheoryProxy::explainPropagation(SatLiteral l, SatClause& explanation) {
   TrustNode tte = d_theoryEngine->getExplanation(lNode);
   Node theoryExplanation = tte.getNode();
   Assert(!d_env.isTheoryProofProducing() || tte.getGenerator());
+  if (isOutputOn(OutputTag::LEMMAS))
+  {
+    output(OutputTag::LEMMAS) << "(lemma ";
```

Comment:
I think you want to output `tte.getProven()` here, otherwise you are outputting the explaination, not the overall lemma.